drivers/cc110x/cc110x-defaultsettings.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2013 INRIA
+ *               2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   drivers_cc110x
+ * @{
+ *
+ * @file
+ * @brief   TI Chipcon CC110x default settings
+ *
+ * @author    Thomas Hillebrandt <hillebra@inf.fu-berlin.de>
+ * @author    Heiko Will <hwill@inf.fu-berlin.de>
+ * @author    Oliver Hahm <oliver.hahm@inria.fr>
+ * @author    Kaspar Schleiser <kaspar@schleiser.de>
+ * @}
+ */
+
+#include "board.h"
+#include "cc110x.h"
+
+/**
+ * @brief   PATABLE with available output powers
+ * @note    If changed in size, adjust MAX_OUTPUT_POWER definition
+ *          in CC110x interface
+*/
+const char cc110x_default_pa_table[8] = {
+              0x00,         ///< -52 dBm
+              0x0D,         ///< -20 dBm
+              0x34,         ///< -10 dBm
+              0x57,         ///< - 5 dBm
+              0x8E,         ///<   0 dBm
+              0x85,         ///< + 5 dBm
+              0xCC,         ///< + 7 dBm
+              0xC3          ///< +10 dBm
+};
+
+const char cc110x_default_base_freq[3] = { 0x21, 0x71, 0x7F };
+
+/**
+ * @brief cc110x default settings
+ */
+const char cc110x_default_conf[] = {
+    0x06, /* IOCFG2 */
+    0x2E, /* IOCFG1 */
+#ifdef CC110X_IOCONF0_VAL
```

Comment:
One board I had to do a port for uses the cc110x as clock source, so SPI&cc1101 get initialized in the board config. IOCONF0 is set to output a clock. If later at driver initialization the cc110x gets reset or ioconf0 overwritten, the board hangs.

I'll add a comment.